tests/saw/s2n_handshake_io.saw

```diff
@@ -372,11 +373,11 @@ let s2n_handshake_io_lowlevel = do {
 
 
     print "Proving correctness of get_auth_type";
-    auth_type_proof <- crucible_llvm_verify llvm "s2n_connection_get_client_auth_type" dependencies true s2n_connection_get_client_auth_type_spec (do {simplify (addsimp equalNat_ite basic_ss); yices;});
```

Comment:
What does this change from true to false do?

Reply:
It changes whether the verifier enables a feature (path satisfiability checking) that is usually not necessary but occasionally may make proofs succeed that would otherwise run forever. In recent SAW changes, it now requires an external tool, and it isn't needed for this proof. So turning it off is simpler.